packages/react/react/src/hooks/useLazyImage.ts

```diff
@@ -0,0 +1,67 @@
+import { useState } from 'react';
+import { useRefEffect } from './useRefEffect';
+
+interface Props {
+  src: string;
+  threshold?: number | number[];
+  root?: Document | Element | null;
+  rootMargin?: string;
+  onInView?: () => void;
+}
+
+/** @tossdocs-ignore */
+export function useLazyImage({ src, rootMargin, threshold, root, onInView }: Props) {
+  const [isLoading, setIsLoading] = useState<boolean>(true);
+
+  const ref = useRefEffect<HTMLImageElement>(element => {
+    if (typeof IntersectionObserver === 'undefined') {
+      element.src = src;
```

Comment:
`isLoading` should be set in this block.

Reply:
I missed it... Thanks for the detailed review.